sale_timesheet_order_line_sync/tests/test_sale_timesheet_order_line_sync.py

```diff
@@ -121,14 +123,14 @@ def test_sale_timesheet_order_line_sync(self):
         self.assertEqual(task.billable_type, "task_rate")
         self.assertAlmostEqual(sale_order_line.qty_delivered, 1.0, 2)
 
-        sale_order2 = self.SudoSaleOrder.create(
+        sale_order2 = self.sudo_sale_order.create(
             {
                 "partner_id": partner.id,
                 "partner_invoice_id": partner.id,
                 "partner_shipping_id": partner.id,
             }
         )
-        sale_order_line2 = self.SudoSaleOrderLine.create(
+        sale_order_line2 = self.sudo_sale_order_line.create(
             {
                 "order_id": sale_order.id,
                 "name": product.name,
```

Comment:
IMO although not appear in Guidelines, uppercase in variables related to models are allowed ant all these changes aren't necessary.

Reply:
Yes, please reduce the diff at maximum

Reply:
Ok, thanks for the suggestions. Changes made-